cmd/githubgen/main.go

```diff
@@ -206,3 +245,44 @@ LOOP:
 
 	return nil
 }
+
+func inList(list []string, id string) bool {
+	for _, m := range list {
+		if id == m {
+			return true
+		}
+	}
+	return false
+}
+
+func getGithubMembers() ([]string, error) {
+	client := github.NewTokenClient(context.Background(), os.Getenv("GITHUB_TOKEN"))
```

Comment:
Is it worth passing these values in?

Reply:
yes, otherwise it fails because the github token is not set.